src/Polly.Core/Outcome.cs

```diff
@@ -1,5 +1,6 @@
 #pragma warning disable CA1815 // Override equals and operator equals on value types
 
+using System;
```

Comment:
Needed?

Reply:
Nope, but I moved this class to TestUtils